src/github-api.ts

```diff
@@ -143,6 +144,7 @@ interface Report {
 }
 
 export async function fetchOtherRunStatus(
+  apiUrl: string,
   token: string,
   params: Parameters<typeof getCheckRunSummaries>[1],
```

Comment:
```suggestion
  params: Parameters<typeof getCheckRunSummaries>[2],
```

Looks like missing to update here is the cause of current typecheck failures

Reply:
Thank you, applied!